src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -23,6 +23,16 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
   @ViewChildren('phasesplitselect')
   components: QueryList<SelectphaseComponent>;
 
+  /**
+   * Leaderboard Precision Value
+   */
+  LeaderboardPrecisionValue = 2;
+
+    /**
+   * Challenge phase split pk Value
+   */
+  challenge_phase_Split_pk;
```

Comment:
Any default value ?

Reply:
yes, by default the leaderboard precision value is set to 2. So i am showing `2` as the default value in the slider.